csgo_bot.py

```diff
@@ -21,7 +19,7 @@
 
 # test -1001280311618
 # CSGO -1001107551770
-GROUP_ID = -1001107551770
+GROUP_ID = -319789223
```

Comment:
It's better if we keep the csgo group.

Reply:
i cant debug it in csgo group , ill post it with the csgo group in the oncoming commits